packages/hub/services/scheduled-payments/executor.ts

```diff
@@ -0,0 +1,157 @@
+import { inject } from '@cardstack/di';
+import { ScheduledPayment } from '@prisma/client';
+import { isBefore, subDays } from 'date-fns';
+import shortUUID from 'short-uuid';
+import { nowUtc } from '../../utils/dates';
+import config from 'config';
+import { Wallet } from 'ethers';
+import { JsonRpcProvider } from '@cardstack/cardpay-sdk';
+
+export let supportedChains = [
```

Comment:
Should this go to the SDK?

Reply:
Oh, I see we already have network related configs in https://github.com/cardstack/cardstack/blob/main/packages/cardpay-sdk/sdk/constants.ts

I'll add the rpc urls there.